RiotSwiftUI/Modules/Room/PollHistory/PollHistoryViewModel.swift

```diff
@@ -14,18 +14,15 @@
 // limitations under the License.
 //
 
+import Combine
 import SwiftUI
 
 typealias PollHistoryViewModelType = StateStoreViewModel<PollHistoryViewState, PollHistoryViewAction>
 
 final class PollHistoryViewModel: PollHistoryViewModelType, PollHistoryViewModelProtocol {
     private let pollService: PollHistoryServiceProtocol
-    private var polls: [PollListData] = []
-    private var fetchingTask: Task<Void, Error>? {
-        didSet {
-            oldValue?.cancel()
-        }
-    }
+    private var polls: [TimelinePollDetails]?
+    private var subcriptions: Set<AnyCancellable> = .init()
```

Comment:
Super minor nit, but could we call this `cancellables` for consistency?

Reply:
Unfortunately `cancellables` is a stored property already present in the superclass.
If I reuse that I risk to mess up the subscription the superclass is adding when I call `removeAll` on that. :-/

Reply:
Derp, my bad! Ignore this one :)